support/hololens/ServoApp/BrowserPage.cpp

```diff
@@ -71,6 +71,10 @@ void BrowserPage::BindServoEvents() {
                 ? Visibility::Collapsed
                 : Visibility::Visible);
       });
+  Window::Current().VisibilityChanged(
```

Comment:
I assume this gets called when we enter immersive mode on the hololens as well, right? Is there an FPS improvement?

Reply:
> I assume this gets called when we enter immersive mode on the hololens as well, right?

Yes.

> Is there an FPS improvement?

I haven't looked.

Reply:
I don't see obvious improvements, but the emulator is probably not the best way of measure that.